vms/platformvm/warp/benchmarks/benchmarks_test.go

```diff
@@ -0,0 +1,188 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package warpbench
```

Comment:
this benchmark is in a different package to avoid an import cycle which I was not able to solve differently. Ideally we'd like this in warp, hopefully it's not too bad

Reply:
Since we are isolating `GetCanonicalValidatorSet` from the `GetValidatorSet` benchmarks, can we get rid of this package and move the `GetCanonicalValidatorSet` benchmark back into the warp package and the specific `GetValidatorSet` benchmarks into the platformvm package?